_posts/2024-08-07-Introducing-a-neural-sparse-two-phase-algorithm.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title:  Introducing the neural sparse two-phase algorithm
+authors:
+  - zhichaog
+  - yych
+  - congguan
+date: 2024-08-07
+categories:
+    - technical-posts
+has_science_table: true
+meta_keywords: OpenSearch semantic search, neural sparse search, semantic sparse retrieval
+meta_description: Introducing a new search pipeline that can reduce the latency of OpenSearch neural sparse search.
```

Comment:
Please update the meta with the following: 

_meta_keywords:_ `neural sparse search, OpenSearch semantic search, neural sparse two phase processor`

_meta_description:_ `OpenSearch continues to refine neural sparse search and retrieval with a two-phase algorithm that significantly reduces computational load while maintaining the quality of the final ranking.
`